lib/client.ml

```diff
@@ -141,8 +349,26 @@ let init ~state ?(router = fun m -> m) ~msg_handler (address, port) =
   let socket = Mutex.create socket in
   let state = Mutex.create (ref state) in
   let inbox = Inbox.create () in
+  let failure_detector =
+    Failure_detector.make
+      { protocol_period = 9; round_trip_time = 3; peers_to_ping = 3 } in
+  let peers =
+    Base.Hashtbl.create ~growth_allowed:true ~size:0 (module Address) in
+  init_peers
+  |> List.map (fun addr ->
+         Base.Hashtbl.add peers ~key:addr ~data:(Peer.from addr))
+  |> ignore;
   let client =
-    ref { address; current_request_id; request_table; socket; state; inbox }
-  in
+    ref
```

Comment:
What about rewriting to:
```ocaml
let client =
  ref
    {
      address = Address.create address port;
      current_request_id = Mutex.create (ref 0);
[...]
```
That i just syntax comment, but this module is already so long... Would be nice to save a few lines which are not really useful to understand, what do you think?